<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchFretEfficiencyDialog</class>
 <widget class="QDialog" name="BatchFretEfficiencyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add multiple FRET efficiency evaluators</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>name template:</string>
       </property>
       <property name="buddy">
        <cstring>nameTemplateEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameTemplateEdit">
       <property name="text">
        <string>{lp1}_{lp2}</string>
       </property>
       <property name="placeholderText">
        <string>{lp1}_{lp2}</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>FÃ¶rster radius:</string>
       </property>
       <property name="buddy">
        <cstring>r0SpinBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="r0SpinBox">
       <property name="value">
        <double>52.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Labeling Positions from:</string>
       </property>
       <property name="buddy">
        <cstring>lpList1</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="allFrom">
       <property name="text">
        <string>all positions</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="lpList1">
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Labeling Positions to:</string>
       </property>
       <property name="buddy">
        <cstring>lpList2</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="allTo">
       <property name="text">
        <string>all positions</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="lpList2">
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BatchFretEfficiencyDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>355</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BatchFretEfficiencyDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>355</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>allTo</sender>
   <signal>stateChanged(int)</signal>
   <receiver>BatchFretEfficiencyDialog</receiver>
   <slot>toggleAllTo(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>allFrom</sender>
   <signal>stateChanged(int)</signal>
   <receiver>BatchFretEfficiencyDialog</receiver>
   <slot>toggleAllFrom(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>toggleAllFrom(int)</slot>
  <slot>toggleAllTo(int)</slot>
 </slots>
</ui>
